course4/homework1/vector2d.h

```diff
@@ -1,9 +1,68 @@
-#pragma once
-
-#include <ostream>
-
-struct Vector2d
-{
-	// TODO
-};
-
+#pragma once
+
+#include <ostream>
+#include <cmath>
+
+struct PolarCoords2d;
+
+struct CartesianCoords2d
+{
+	double x, y;
+	
+	CartesianCoords2d(double x, double y) : x(x), y(y) {}

```

Comment:
Since the struct is POD, you don't really need the constructor, you can just create instance using list initialization:
```CartesianCoords2d{1.2, 3.4}```